src/aura/cmp1/cmp1Controller.js

```diff
@@ -0,0 +1,16 @@
+({
+    fireApplicationEvent : function(cmp, event) {
+        
+        var accName = cmp.get("v.paccSearch");
+        
+        if(accName.length > 2) {
+            if(event.getParams().keyCode == 13){
```

Comment:
why here use a nested if ? and not just simply an && operator?
`If(accName.lenght >2 && event.getParams().keyCode == 13)`
Also what will happen with this approach in mobile ? I know it's not a requirement, but it's always a good practice to think in all the scenarios.